common/components/common/projects/ProjectVolunteerButton.jsx

```diff
@@ -151,28 +152,36 @@ class ProjectVolunteerButton extends React.PureComponent<Props, State> {
   }
 
   _renderVolunteerButton(): React$Node {
-    return this.state.isAlreadyVolunteering ? (
+    if (this.state.isAlreadyVolunteering) {
       // TODO: Make this its own component and hook up to My Projects page
-      <Button
+      return (<Button
         className="AboutProject-button"
         type="button"
         variant="destructive"
         onClick={this.handleShowLeaveModal}
       >
         Leave Project
-      </Button>
-    ) : (
-      <Button
-        variant="primary"
-        className="AboutProject-button"
-        type="button"
-        disabled={this.state.buttonDisabled}
-        title={this.state.buttonTitle}
-        onClick={this.handleShowJoinModal}
-      >
-        Volunteer With Project
-      </Button>
-    );
+      </Button>);
+
+    } else {
+        if (!_.isEmpty(this.state.project.project_positions)) {
```

Comment:
Didn't call this out in the acceptance criteria, but we would also like to hide the button in the case that all positions are hidden (`project_position.isHidden === true`)

Reply:
> Didn't call this out in the acceptance criteria, but we would also like to hide the button in the case that all positions are hidden (`project_position.isHidden === true`)

will do